recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,2 @@
-MACOSX_SDK_VERSION:            # [osx and x86_64]
-  - 10.15                      # [osx and x86_64]
+#MACOSX_SDK_VERSION:            # [osx and x86_64]
+#  - 10.15                      # [osx and x86_64]
```

Comment:
MACOSX_SDK 10.15 is available already

Reply:
when I uncomment those build fails on OSX

Reply:
then remove thos lines completely IMHO